packages/storybook-html/src/patterns/form-validation/examples/password-reset.scss

```diff
@@ -108,7 +108,7 @@
   }
 }
 
-.form__inline-error-message {
+.inline-error-message {
```

Comment:
Should this not be a modifier class 

.error-summary
.error-summary--inline

Reply:
Sorry just realized  error-summary is the overall report not the line item. 

Still think you might consider a name change as the inline is implied as a changeable attribute 

.error-message
.error-message--inline

.error
.error--inline

.input-error
.input-error--inline

Reply:
'Inline' isn't a variant but aligns with existing form validation vernacular. I.e. error summary (outside the form contents) vs. inline  messages (within the form contents). I will change to `error-message` just to simplify anyway.